lib/util/http/github.ts

```diff
@@ -205,6 +205,10 @@ export class GithubHttp extends Http<GithubHttpOptions, GithubHttpOptions> {
 
       // istanbul ignore else: Can result be null ???
       if (result !== null) {
+        // Adjust the response if request has been made with Github App
+        if (result.body.repositories) {
+          result.body = result.body.repositories;
+        }
```

Comment:
This is not appropriate for this file as it should not be field-aware

Reply:
Thanks for the feedback. You are right, and I also realized that it was not necessary because I can use the paginationField to achieve the same result.

Reply:
Thanks for the feedback. You are right, and I also realized that it was not necessary because I can use the paginationField to achieve the same result.